src/Disks/IDiskRemote.cpp

```diff
@@ -270,7 +271,7 @@ std::unordered_map<String, String> IDiskRemote::getSerializedMetadata(const std:
     return metadatas;
 }
 
-void IDiskRemote::removeMetadata(const String & path, std::vector<std::string> & paths_to_remove)
+void IDiskRemote::removeMetadata(const String & path, std::unordered_map<String, std::vector<String>> & paths_to_remove)
```

Comment:
unordered map looks too complicated.
Looks like this functions only changes `paths_to_remove[path]`, why not to use std::vector as before?